README.md

```diff
@@ -5,48 +5,31 @@
+## Libraries based on `poker-base`
 
-## Current State of Library - What Needs to be Done
-
-Currently the library is going through a final revision phase, in order to make sure that the current API is stable and maintainable. Some major issues (for which PRs or help would be greatly welcomed) include:
- - Use a more optimal representation of the Card type, such as in https://github.com/ghais/poker. If you are ghais - let's work together :) Many type declarations in this library should use UNPACK pragmas or more efficient representations (for example, `data Suit = Suit Bit Bit`???).
- - Continue to iterate on documentation.
- - Improve some APIs on types. Some usage of private type constructors are probably misuses, for example. Some creator-functions are also either not-yet-revised or untested.
- - Avoid using Enum as a fake attempt at optimisation. I'm only lying to myself.
- - Documentation with reasonable examples
-
-## Usage in Other Libraries
-
-I use this library in a couple different little projects I work on at home. Some projects that I'll be open-sourcing in the near future include:
- - poker-game: a library for the rules of poker
- - poker-history: a library for parsing poker site hand histories. I currently support PokerStars and Bovada - if you have any hand databases for other sites, I will happily support them.
- - poker-reflex: a collection of different [reflex-frp](https://reflex-frp.org/) JS widgets for stuff like showing a range or poker table with statefulness.
-
-I also use this library for some private work that I use to study poker, when I get the chance, which is not often :(, but I'm always excited to talk about it!
-
-## Let's Collaborate!
-
-I have a bunch of work that I've worked on on-and-off for a while now and I figure now's the time to get it all out there! Some stuff I am interested in:
- - abstracting GTO poker solvers' solutions (using ML and other techniques).
- - UI-based DSLs and Haskell DSLs for querying poker game trees, especially against databases of online hands.
- - probably anything poker-related if you get me thinking about it LOL
+ - [`poker-game`](https://github.com/santiweight/poker-game): a library for the rules of no-limit poker (not just holdem)
+ - [`poker-histories`](https://github.com/santiweight/poker-histories): a library for parsing poker site hand histories. Currently supports PokerStars and Bovada.
+ - [`poker-fold`](https://github.com/tonyday567/poker-fold): TODO @Tony
```

Comment:
Name change pending: I am torn between `poker-ai` (assisted intelligence), and `phoker` (obvious reasons).

: a experimental platform for researching poker strategy and intelligence.